api/README.md

```diff
@@ -21,48 +19,39 @@ The latter can be installed with the following command
 cargo install sqlx-cli --no-default-features --features rustls,postgres
 ```
 
-### Setup
-
-Initialize a database for development by spawning a docker container with
-
-```bash
-./scripts/init_db.sh
-```
-
-### Run
-
-Start the application.
+### Quickstart
 
 ```bash
-cargo run
+# Start the docker-compose development composition
+make run
+# Run tests
+carg test
```

Comment:
cargo

Reply:
Addressed in https://github.com/swissgeol/ngm/pull/827/commits/64d2e491ea8b53a42bd1f8ab13bee9b45c89a357